tests/topology/test_pbc_cell_div.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
```

Comment:
The test file should be structured like the others in the tests directory: a series of single test_smthg argument-less functions, each testing an execution scenario:
* Make a division with pbc if the cell is not on the border
* make a division with pbc if the cell is not on the border

Reply:
So you can remove all the prints and gradient descent code, and create 3 functions (one boundary, both boundaries, no boundary). In order to make the tests effective, instead of a gradient descent, you can use the validate() method:
```
assert sheet.validate()
```
This should make sure that we didn't make holes and stuff like that

You can also check that the number of faces has increased by 1 post division.